src/dropin-model.js

```diff
@@ -50,13 +63,30 @@ EventEmitter.createChild(DropinModel);
 
 DropinModel.prototype.initialize = function () {
   var self = this;
+  var dependencyReadyInterval = setInterval(function () {
+    var ready = true;
+
+    ASYNC_DEPENDENCIES.forEach(function (dep) {
```

Comment:
Do we need to iterate through all possible `ASYNC_DEPENDENCIES` here, or can we only iterate the filtered `self.dependencyStates` array?

Reply:
You're totally right. Let's do that. 

```suggestion
    Object.keys(self.dependencyStates).forEach(function (dep) {
```